lib/cdo/shared_constants.rb

```diff
@@ -0,0 +1,16 @@
+# A set of constants shared between ruby and JS via /shared/config/sharedConstants.json
```

Comment:
Put this in lib/cdo since we seemed to have other sets of constants here. Wonder if this AND the json file should live here?